Align on a README

annevk opened the pull request:
This doesn't cover all standards (Compatibility, Console, File System, Fullscreen, HTML, and Streams are excluded for now), but hopefully we can get there in incremental steps.

domenic commented on README.md.template:
```diff
@@ -0,0 +1,82 @@
+
+## Building "locally"
+
+For quick local iteration, run `make`. To verify your changes locally, run `make deploy`. See more in the [WHATWG Contributor Guidelines](https://github.com/whatwg/meta/blob/main/CONTRIBUTING.md#building).
```
```suggestion
For quick local iteration, run `make`; this will use a web service to build the standard, so that you don't have to install anything. See more in the [WHATWG Contributor Guidelines](https://github.com/whatwg/meta/blob/main/CONTRIBUTING.md#building).
```

domenic commented on README.md.template:
```diff
@@ -0,0 +1,82 @@
+
+For quick local iteration, run `make`. To verify your changes locally, run `make deploy`. See more in the [WHATWG Contributor Guidelines](https://github.com/whatwg/meta/blob/main/CONTRIBUTING.md#building).
+
+## Formatting
```
I know these are your formatting preferences but I'd prefer not making them the universal default, without more discussion. E.g. I and other editors prefer using much more Markdown syntax.

annevk commented:
I think this better addresses the remaining feedback.

The idea is that you can have a shared README section across specifications that hopefully eventually we can standardize upon somehow (one way or another) and each specification also gets a READMEEND section they maintain locally and that is pulled in when doing a spec-factory deploy.

annevk commented:
I guess "README-addition-anne" needs to be called out in https://github.com/whatwg/spec-factory/blob/main/README.md as it doesn't end in `.template`. Perhaps we should call them something along the lines of `README.md.anne.template-part` and call out `.template-part` even though it doesn't really have any particular meaning beyond that?

tabatkins commented on README-addition-anne:
````diff
@@ -0,0 +1,50 @@
+```
+is.
+
+End tags may be included (if done consistently) and attributes may be quoted (using double quotes), though the prevelant theme is to omit end tags and not quote attributes (unless they contain a space).
````
```suggestion
End tags may be included (if done consistently) and attributes may be quoted (using double quotes), though the prevalant theme is to omit end tags and not quote attributes (unless they contain a space).
```

domenic reviewed (approved):
This seems like it'll work nicely.

I like the idea of using a different extension like `.template-part`.
